vspace/src/main/java/edu/asu/diging/vspace/references/ReferenceDisplayDefault.java

```diff
@@ -0,0 +1,93 @@
+package edu.asu.diging.vspace.references;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.model.impl.Reference;
+
+@Service
+public class ReferenceDisplayDefault implements ReferenceDisplayProvider{
+    
+    public String getReferenceDisplayText(Reference reference) {
+        
+        String referenceDisplayText = "";
+        String title = reference.getTitle();
+        String author = reference.getAuthor();
+        String year = reference.getYear();
+        String journal = reference.getJournal();
+        String url = reference.getUrl();
+        String volume = reference.getVolume();
+        String issue = reference.getIssue();
+        String pages = reference.getPages();
+        
+        referenceDisplayText += getRefAuthorDisplayText(author) + getRefYearDisplayText(year) + getRefTitleDisplayText(title)
+            + getRefJournalDisplayText(journal) + getRefPagesDisplayText(pages) + getRefVolumeDisplayText(volume) + getRefIssueDisplayText(issue)
+            + getRefUrlDisplayText(url);
```

Comment:
I think I would use method chaining here so it can be reused, meaning:
- Create a class that has methods like `addAuthors`, `addYear`, etc.
- each methods returns `this`
- some methods could have parameters like which separators (for authors) to use or if the value should be surrounded with parentheses (e.g. year)
- this here, you would just call:
```
myFormatter.addAuthors(author).addYear(year,  true).addTitle(title). ...  .asString();
```
This way, if we want to add another format, the class can just be reused to change for example the order or make other modifications.

Does that make sense?